spec/models/initiative_spec.rb

```diff
@@ -0,0 +1,75 @@
+# == Schema Information
+#
+# Table name: initiatives
+#
+#  id          :integer          not null, primary key
+#  description :string
+#  startdate   :date
+#  status      :integer
+#  title       :string           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  parent_id   :integer
+#
+# Indexes
+#
+#  index_initiatives_on_parent_id  (parent_id)
+#
+# Foreign Keys
+#
+#  parent_id  (parent_id => initiatives.id)
+#
+require 'rails_helper'
+
+RSpec.describe Initiative, type: :model do
+  context 'in its entities,' do
+      it 'may have helpers' do
+        source = Wyeworker.new(name: "Source Pablo")
+        initiative_no_helpers = Initiative.new(title: "Test Inititative No Helpers", source: source)
```

Comment:
Highly recommend the usage of factory bot

Reply:
Oh, I checked it out, agreed!